package.json

```diff
@@ -21,6 +21,7 @@
     "karma": "^0.12.16",
     "karma-chrome-launcher": "^0.1.3",
     "karma-coverage": "^0.2.4",
+    "karma-ie-launcher": "^0.2.0",
```

Comment:
please remove karma-ie-launcher (you can manually test it in IE, rather than launch it automatically)